cmd/server.go

```diff
@@ -12,6 +12,9 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/TierMobility/boring-registry/pkg/storage"
```

Comment:
Maybe group this import together with the rest of `github.com/TierMobility/boring-registry/pkg/*`?